### ISSUE TITLE
Conversion of a Keras TCN to PyTorch-tcn

### ISSUE THREAD

**Issue by myliheik** [paul-krug/pytorch-tcn#26]
I have used TCN in Keras implementation as follows:

`keras_model = TCN(input_shape=(None, xtrain3d.shape[2]), nb_filters = 32, padding = 'causal', kernel_size = 2,  nb_stacks = 1, dilations = [1, 2, 4, 8], return_sequences = True)
`

I am unsure about the conversion to PyTorch-tcn. Is the architecture same in PyTorch as follows:

`pytorch_model = TCN(num_inputs = xtrain3d.shape[2], num_channels = [32, 32, 32, 32], kernel_size  = 2, dilations = [1, 2, 4, 8], input_shape = 'NLC', causal = True).to(device)`
